src/scss/06_components/paragraphs/_contact-card.scss

```diff
@@ -0,0 +1,78 @@
+.contact-card {
+  display: flex;
+}
+
+.contact-card--with-image {
+  @include breakpoint($breakpoint-s) {
+    flex-direction: column;
+  }
+}
+
+.contact-card--information-container {
+  @include breakpoint($breakpoint-s) {
+    max-width: 200px;
+  }
+}
+
+.contact-card--person-social-media {
+  display: flex;
+
+  .hel-icon {
+    height: 26px;
+    margin-right: $spacing-half;
+    width: 26px;
+  }
+}
+
+.contact-card--image-container {
+  margin-bottom: $spacing;
+  margin-right: $spacing;
+
+  @include breakpoint($breakpoint-s) {
+    margin-right: 0;
+  }
+
+  img {
+    height: 100px;
+    width: 100px;
+
+    @include breakpoint($breakpoint-s) {
+      height: 200px;
+      width: 200px;
+    }
+  }
+}
+
+.contact-card--person-name {
+  font-size: 20px;
```

Comment:
We should not have font-size or other font-related code in our SCSS, we should use `@include font('special.navigation.icon_label');`-font styles and if something does not match what you see in design, ask from Sauli what font style should be used.

Reply:
Check! I'll change these